activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-api/src/main/java/org/activiti/cloud/services/rest/api/AdminController.java

```diff
@@ -0,0 +1,17 @@
+package org.activiti.cloud.services.rest.api;
+
+import org.activiti.cloud.services.api.model.Task;
+import org.activiti.cloud.services.rest.api.resources.TaskResource;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PagedResourcesAssembler;
+import org.springframework.hateoas.PagedResources;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@RequestMapping(value = "/admin/v1")
+public interface AdminController {
+
+    @RequestMapping(value = "/tasks", method = RequestMethod.GET)
```

Comment:
I discovered recently that a `@GetMapping` annotation is also available, this save the `method = RequestMethod.GET`, but not need to use it here.

Reply:
Yeah the other controllers are doing RequestMethod.GET, best to leave all or change all

Reply:
👍